docs/developer-resources/walkthroughs/no-code-erc20.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Deploy an ERC20 token to Celo'
+---
+
+You can easily deploy token contracts that use the ERC20, ERC721, or ERC1155 token standard to Celo without having to write any code.
+
+# Deploying ERC20 Tokens on Celo (no code)
+
+In this tutorial, we will go over how to deploy an ERC20 token contract. The process is very similar for deploying other tokens as well.
+
+1. Install [Metamask](https://metamask.io/).
+2. [Add the Celo network](/getting-started/wallets/using-metamask-with-celo/manual-setup#adding-a-celo-network-to-metamask) to Metamask. I suggest adding the Alfajores testnet to Metamask as well, so you can test contract deployments before deploying to mainnet.
```

Comment:
I'd change "I suggest" to "We suggest" given you are using I and We interchangeably in the doc.